readme.md

```diff
@@ -131,7 +132,6 @@ Demos might work only on Chrome. Firefox implementation is not complete.
 - [WebGPU Metaballs](https://toji.github.io/webgpu-metaballs/) - by [Brandon Jones](https://github.com/toji) - [repository](https://github.com/toji/webgpu-metaballs)
 - [WebGPU External Texture Test](https://toji.github.io/webgpu-external-test/) - by [Brandon Jones](https://github.com/toji) - [repository](https://github.com/toji/webgpu-external-test)
 - [Online WGSK Editor](https://takahirox.github.io/online-wgsl-editor/) - by [Takahiro](https://github.com/takahirox) - [repository](https://github.com/takahirox/online-wgsl-editor)
-- [Safari WebKit demos](https://webkit.org/demos/webgpu/)
```

Comment:
@munrocket why did delete this line ?

Reply:
They removed support of this yearly days demos in Safari TP. It is literally impossible to check anywhere.